### PR TITLE
Typo fix and add a link to Breadcrumbs plugin on showcase note

### PR DESCRIPTION
**Pull request by lkadre**
- lkadre
- Breadcrumbs plugin showcase

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [x] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

File: 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ publish: true
 
 *Note: this is a refined version of the post by [[lkadre]] on [this discussion](https://github.com/SkepticMystic/breadcrumbs/discussions/175) on Github about [[How to get the most out of the Breadcrumbs plugin]].*
 
-#### Example: Concept notes (Law)
+Since the [[breadcrumbs|Breadcrumbs]] plugin is so powerful, it can be useful to see an showcase for how other people use it to maximise creativity and analysis. 
+
+This is specifically about my notes on Law, which I acknowledge is probably a very niche topic but hopefully it can give you some ideas of how to set up your hierarchies and especially how to work with notes in multiple hierarchies. You don't have to choose one!
 
-This is specifically about my notes on Law, which I acknowledge is probably very niche but hopefully it can give you some ideas of how to set up your hierarchies and especially how to work with notes in multiple hierarchies. You don't have to choose one!
+#### Example: Concept notes (Law)
```
Comment (chrisgrieser) [inline 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md:16]: I don't really get the purpose of this heading. To me, it does not make sense to have a subheading, when there is no other subheading 😅 

I'd suggest either removing the subheading or adding a second one somewhere.

**Review by chrisgrieser — COMMENTED** [review]
I only fixed some minor language

File: 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md (hunk 1 above)
Comment (argenos) [reply 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md:16]: I don't mind it for navigation purposes. Click on the TOC on the right to jump to the example

**Review by argenos — APPROVED** [review]

File: 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md (hunk 1 above)
Comment (chrisgrieser) [reply 03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md:16]: ah, haven't thought of that use case. makes sense!

**Pull request merged**